### DISCUSSION TITLE
Grafana+Prometheus: Windows:Looking for the query to fetch Top10 processes in windows to display

### DISCUSSION THREAD

**Discussion by sravanes** [grafana/grafana#56079] [Dashboards, Panels & Visualizations]
Hi All - I am using Grafana +prometheus agent to fetch data on the windows server

Objective: Needs to display in Grafana dashboard,  to fetch Top 10 Processes in the windows and looking for the query.

Data used:

I used following query to fetch "number of processes" which was successful ,but looking for top 10 processes in that one.

windows_os_processes{job=~“$job”,instance=~“$instance”}

![image](https://user-images.githubusercontent.com/112639426/193252349-722876d6-47d3-4586-a276-0e332f750b77.png)

**Comment by sravanes** [comment]
Hi All - Can some one repsond to this query or can you please find the proper contacts to reach out them.

**Reply by sravanes** [reply]
Succeded with following query on windows

topk(10,irate(windows_process_cpu_time_total{instance=~"$instance"}[3m]))

But problem is ,its showing all processes instead top10

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
